### ISSUE TITLE
TypeError: bank[i] is null

### ISSUE THREAD

**Issue by Arcinsanity** [MelvorIdle/melvoridle.github.io#108]
**Describe the bug**
Was fighting Mythril knights using magic, received the following error after the game became unresponsive:
TypeError: bank[i] is null 
This is the 3rd time my bank has gotten a "null" item every time i have been doing something different. And each time Malcs has had to manually fix my save. 
**To Reproduce**
Unknown cause.
Edit: The only common thing i can think of is that each time this error has occurred the game had been on for an extended period of time without refreshing, reloading, etc. (over 20 hours) 
**Expected behavior**
N/A
**Screenshots**
N/A

**Browser**
Please tell us what browser you are using and what version.
Fire Fox latest version.
**Console output**
Please open the console by pressing F12 and copy it here.
TypeError: bank[i] is null firemaking.js:302:7
    updateAvailableLogs https://melvoridle.com/assets/js/game/firemaking.js?200:302
    updateSkillWindow https://melvoridle.com/assets/js/game/main.js?203:1203
    updateWindow https://melvoridle.com/assets/js/game/main.js?203:945
    loadGame https://melvoridle.com/assets/js/game/save.js?200:310
    initiateLoad https://melvoridle.com/assets/js/game/save.js?200:376
    onload https://melvoridle.com/assets/js/game/save.js?200:408

My save file: [melvoridlesave.txt](https://github.com/MelvorIdle/melvoridle.github.io/files/4269656/melvoridlesave.txt)

Edit Some more info: once this error happens the game will no longer load so the save must be fixed via the console or completely deleted.

**Comment by zhs2** [comment]
Bump. I've also run into this exact same bug, but I'm unsure of where to look for a save file to upload. I'm using Waterfox 2020.02.1. This happened after I reached a point in the game where clicking any action would cause the action's bar to begin increasing but once it fills a single time it stops and no items are given, whereupon I attempted to reload the page to fix it (which has worked once before) and ran into this.
